lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -368,15 +368,17 @@ def run
   def wait_for_access(connect_host, connect_port, protocol)
     if protocol == 'winrm'
       load_winrm_deps
+      if get_config(:winrm_transport) == 'ssl' && get_config(:winrm_port) == '5985'
+        config[:winrm_port] = '5986'
```

Comment:
Except they added that to 1.0.0 and it didn't work with our code and I didn't have time to debug why. Not to mention they don't have an official release of 1.0.0 yet so this code is essentially a temporary workaround.